tljh/user_creating_spawner.py

```diff
@@ -2,8 +2,9 @@
 from tljh import user
 from systemdspawner import SystemdSpawner
 from traitlets import Dict, Unicode, List
+from jupyterhub_configurator.mixins import ConfiguratorSpawnerMixin
 
-class UserCreatingSpawner(SystemdSpawner):
+class UserCreatingSpawner(ConfiguratorSpawnerMixin, SystemdSpawner):
```

Comment:
We should use this mixin only if configurator is enabled - otherwise, any bugs in the configurator backend will stop new user spawns!

Reply:
Good idea. I just added a commit that does this with [`type`](https://docs.python.org/3/library/functions.html?#type) to avoid duplicating code. I tried it within a dev environment and it worked but I'm not sure at all if this is acceptable or if there's a better way.